core/libp2p/discovery/peer_discovery.hpp

```diff
@@ -6,27 +6,23 @@
 #ifndef KAGOME_PEER_DISCOVERY_HPP
 #define KAGOME_PEER_DISCOVERY_HPP
 
-#include <vector>
-
-#include "libp2p/common/peer_info.hpp"
-
 namespace libp2p::discovery {
   /**
    * Continuously scans the network to find and cache new peers
    */
   class PeerDiscovery {
-    /**
-     * Return observable to peers, discovered by this module
-     * @return observable to new peers
-     */
-    // TODO(@warchant): PRE-90 review types, rewrite docs
-    virtual std::vector<common::PeerInfo> getNewPeers() const = 0;
-
-    /**
-     * Return all peers, which were found by this node
-     * @return reference to PeerInfo collection
-     */
-    virtual const std::vector<common::PeerInfo> &getAllPeers() const = 0;
+//    /**
```

Comment:
Is commented code necessary?

Reply:
We don't know the exact interfaces yet, but it's still better to leave those functions at least commented here, as they give some understanding of what this class is supposed to do